.github/workflows/centos7.yml

```diff
@@ -47,28 +76,30 @@ jobs:
       ORTOOLSDIR: ${{ github.workspace }}/or-tools
 
     runs-on: ubuntu-latest
-    needs: docker_publish
+    needs:
+      - docker_publish
+      - doc_generation
     container: 'antaresrte/rte-antares:centos7-simulator'
     strategy:
       matrix:
         ortools: [ON, OFF]
 
     steps:
-           
     - uses: nelonoel/branch-name@v1.0.1
-    
+
     - name: Checkout
       run: |
           git clone $GITHUB_SERVER_URL/$GITHUB_REPOSITORY.git -b ${BRANCH_NAME} .
-    
+
     - name: Get release
       id: get_release
       uses: bruceadams/get-release@v1.2.0
-           
+      if: ${{ env.IS_RELEASE == 'true' }}
+
     - name: Init submodule
       run: |
-           git submodule update --init --recursive src
-           
+          git submodule update --init --recursive src .
+
```

Comment:
Submodule `vcpkg` doesn't need to be initialized in CentOS (not used), only init `src/antares-deps` and `src/tests/resources/Antares_Simulator_Tests`.